diffpy/snmf/tests/test_subroutines.py

```diff
@@ -41,13 +41,35 @@ def test_get_stretched_component(tgso):
     np.testing.assert_allclose(actual, expected, rtol=1e-03)
 
 
-def test_update_weights_matrix():
-    assert False
+tuwm = [
+
+]
 
 
-def test_get_residual_matrix():
+@pytest.mark.parametrize
+def test_update_weights_matrix(tuwm):
     assert False
 
 
-def test_reconstruct_data():
+tgrm = [
+
+]
+
```

Comment:
please don't leave blank lines between these test inputs and the tests (in any of your tests).  It is easier to follow if they are tightly grouped together in my opinion

Reply:
I will fix this